src/sql/src/kafka_util.rs

```diff
@@ -36,11 +38,12 @@ enum ValType {
 
 // Describes Kafka cluster configurations users can suppply using `CREATE
 // SOURCE...WITH (option_list)`.
-// TODO(sploiselle): Support overriding keys, default values.
+// TODO(sploiselle): Support overriding keys.
 struct Config {
     name: &'static str,
     val_type: ValType,
     transform: fn(String) -> String,
+    default: fn() -> Option<String>,
```

Comment:
Random drive-by, but why a `fn -> Option<String>` instead of an `Option<String>`?

Reply:
That's a great question and the answer is that this is vestigial. There's no need to make this a function.

Reply:
This is now an `Option<String>`.